src/providers/dropbox.js

```diff
@@ -0,0 +1,26 @@
+export default (options) => {
+  console.log('> Dropbox')
+  return {
+    id: 'dropbox',
+    name: 'Dropbox',
+    type: 'oauth',
+    version: '2.0',
+    scope: 'account_info.read',
+    params: { grant_type: 'authorization_code' },
+    accessTokenUrl: 'https://api.dropboxapi.com/oauth2/token',
+    authorizationUrl:
+      'https://www.dropbox.com/oauth2/authorize?token_access_type=offline&response_type=code',
+    profileUrl: 'https://api.dropboxapi.com/2/users/get_current_account',
+    profile: (profile) => {
+      return {
+        id: profile.account_id,
+        name: profile.name.display_name,
+        email: profile.login,
+        image: profile.profile_photo_url,
+        email_verified: profile.email_verified
+      }
+    },
+    protection: 'none',
```

Comment:
```suggestion
    protection: ["state", "pkce"]
```

From the docs you linked, it looks like Dropbox supports both state and pkce:

![image](https://user-images.githubusercontent.com/18369201/116532265-56f57680-a8e0-11eb-8a6f-e28111431ee2.png)

I think we should add them by default